kafka-client/src/main/java/dev/responsive/kafka/internal/stores/ResponsiveKeyValueStore.java

```diff
@@ -118,9 +110,7 @@ public void init(final StateStoreContext storeContext, final StateStore root) {
       operations = opsProvider.provide(params, storeContext, taskType);
       log.info("Completed initializing state store");
 
-      storeRegistry = registryProvider.apply(storeContext.appConfigs());
       open = true;
-      operations.register(storeRegistry);
```

Comment:
No need to explicitly register anything now, we just take care of everything when the Operations are created (or closed)